### ISSUE TITLE
Latest update seems not to load version chosen in Plugin dropdown menu

### ISSUE THREAD

**Issue by paulhibbitts** [getgrav/grav-plugin-bootstrapper#43]
Thanks again for updating this plugin with the latest and greatest Bootstrap versions!

After installing and testing the update, I cannot seem to ever change it from v4 loading. After some more testing, here is a repeatable recipe that hopefully you can replicate:

1) Download grav-admin skeleton
2) Install Bootstrap 4 theme
3) Enable Bootstrapper plugin, choose Bootstrap 5
4) Refresh front-end, view source page and v4 is still loaded

Not sure why, but if I change to v5 and save, then leave plugin and return it is back to v4 - might it be some sort of config default/override left over from the earlier version?

Please let me know if any more info is of help etc. Happy to help test too.

Thanks very much,
Paul

**Comment by rhukster** [comment]
I tried with my sandbox setup and changed the version in the admin to `v5` and saved, showed up straight away as loading the 5.0.0-beta3 version.  

```
<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta3/dist/css/bootstrap.min.css?g-ddf0ae7e" type="text/css" rel="stylesheet">
```

and without CDN:

```
<link href="/grav-sandbox/user/plugins/bootstrapper/css/v5/bootstrap.min.css?g-8b24ab2a" type="text/css" rel="stylesheet">
```

Changed it back to `v4` and loaded 4.6.0.

```
<link href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.0/dist/css/bootstrap.min.css?g-84502823" type="text/css" rel="stylesheet">
```

and without CDN:

```
<link href="/grav-sandbox/user/plugins/bootstrapper/css/v4/bootstrap.min.css?g-fe877048" type="text/css" rel="stylesheet">
```

**Comment by paulhibbitts** [comment]
Hi @rhukster , based on your findings I tried to do the same with Quark set as my Active Theme and then everything worked as expected! My testing was with the Bootstrap4 theme which must have over-riden the plugin Bootstrap loader. I see the line `$this->config->set('plugins.bootstrapper.version', 'v4');` in the Bootstrap4 theme so that explains the behavior I was seeing - sorry I did not catch that earlier!
